### PR TITLE
E2E testing

### PR DESCRIPTION
**Pull request by jacogr**
- Move e2e tests to api from api-provider
- Add e2e tests for api-rx

Related #161

### CODE REVIEW

File: packages/api-rx/test/e2e/subscriptions.test.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+// Copyright 2017-2018 @polkadot/api-provider authors & contributors
```
Comment (amaurymartiny) [inline packages/api-rx/test/e2e/subscriptions.test.js]: Incorrect package name

File: packages/api/test/e2e/chain.test.js [hunk 2]
```diff
@@ -2,8 +2,8 @@
 // This software may be modified and distributed under the terms
 // of the ISC license. See the LICENSE file for details.
 
-import Api from '../../api/src';
-import Ws from '../src/ws';
+import Ws from '../../../api-provider/src/ws';
```
Comment (amaurymartiny) [inline packages/api/test/e2e/chain.test.js]: Could use `@polkadot/provider`? At least it's robust if we change folder names.

File: packages/api-rx/test/e2e/subscriptions.test.js (hunk 1 above)
Comment (jacogr) [reply packages/api-rx/test/e2e/subscriptions.test.js]: Dangit, improper move. Will fix.

File: packages/api/test/e2e/chain.test.js (hunk 2 above)
Comment (jacogr) [reply packages/api/test/e2e/chain.test.js]: Yeap, should really do. Will fix.

**Pull request merged**